### PR TITLE
Faster measure perf

### PR DESCRIPTION
**Pull request by FG-TUM**
# Description

- [x] added generator for uniform distribution to md-flex
- [x] changed measurePerf script to use uniform generator
- [x] extended measurePerf sampling points

## ~Resolved Issues~

# How Has This Been Tested?
- [x] executed script lol

### CODE REVIEW

File: examples/md-flexible/CMakeLists.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 file(GLOB_RECURSE MY_SRC
         "*.cpp"
         "*.h"
+        "../../tests/testAutopas/testingHelpers/RandomGenerator.cpp"
```
Comment (SteffenSeckler) [inline examples/md-flexible/CMakeLists.txt]: Eventually we should move this to some proper location, i.e., utils directory of autopas or some extra directory outside of autopas.

Comment (SteffenSeckler) [reply examples/md-flexible/CMakeLists.txt]: and all other generators

File: examples/md-flexible/measurePerf.sh [hunk 2]
```diff
@@ -63,27 +63,30 @@ do
             # iterate over molecules with the correct repetition
             for i in `seq 0 $(( ${#Mols[@]} - 1 ))` ;
             do
+                thisReps=${Reps[$i]}
+                if [[ ${container} = 'DirectSum' && ${Mols[$i]} -ge 2048 ]];
```
Comment (SteffenSeckler) [inline examples/md-flexible/measurePerf.sh]: comment why you do this

Comment (FG-TUM) [reply examples/md-flexible/measurePerf.sh]: done cd1fdb996cce61c3a8cb54fa8e7afb3fa5fd9afb

File: examples/md-flexible/CMakeLists.txt (hunk 1 above)
Comment (FG-TUM) [reply examples/md-flexible/CMakeLists.txt]: #129

**Pull request merged**